packages/utils/src/date/index.ts

```diff
@@ -6,6 +6,7 @@ type DateFnsFormatInput = Date | number | string;
 interface ConversionOptions {
 	timeZone: string,
 	sourceTimeZone?: string,
+	locale?: Object,
```

Comment:
Can we precise the type for `locale` by any chance?

Reply:
this `locale` will be passed to `format` method of date-fns, the type definition is taken from there.
![image](https://user-images.githubusercontent.com/3214228/146900697-58682500-9065-4952-a438-6ea3542269b0.png)

Reply:
you can found Locale type here
https://unpkg.com/date-fns@2.27.0/format/index.js.flow
the index.d.ts makes me 🤔